float32 left_slope
float32 right_slope

bool is_left
bool is_right

float32 bias_from_left
float32 bias_from_right
